crates/ruff_linter/resources/test/fixtures/flake8_bugbear/B006_8.py

```diff
@@ -0,0 +1,20 @@
+def foo(a: list = []):
+    raise NotImplementedError("")
+
+
+def bar(a: dict = {}):
+    """ This one also has a docstring"""
+    raise NotImplementedError("and has some text in here")
+
+
+def baz(a: list = []):
+    """This one raises a different exception"""
+    raise IndexError()
+
+
+def qux(a: list = []):
+    raise NotImplementedError
+
+
+def quux(a: list = []):
+    raise NotImplemented
```

Comment:
What's the purpose of this example?

Reply:
To verify that we treat `raise NotImplemented` like `raise NotImplementedError`. (Just to be clear, these aren't examples, they're tests.)